src/components/shared/pcsGrid/pcsGrid.js

```diff
@@ -8,9 +8,9 @@ import { isFunc } from 'utilities';
 import { Indicator } from '../indicator/indicator';
 import { ROW_HEIGHT } from 'components/shared/pcsGrid/pcsGridConfig';
 
-import '../../../../node_modules/ag-grid/dist/styles/ag-grid.css';
-import '../../../../node_modules/ag-grid/dist/styles/ag-theme-dark.css';
-import './pcsGrid.css';
+import '../../../../node_modules/ag-grid/src/styles/ag-grid.scss';
```

Comment:
Are these actual scss files in the node modules?

Reply:
Yes.